rescan.go

```diff
@@ -0,0 +1,116 @@
+package dcrlibwallet
+
+import (
+	"math"
+	"time"
+
+	"github.com/decred/dcrwallet/errors"
+	w "github.com/decred/dcrwallet/wallet/v3"
+)
+
+func (mw *MultiWallet) RescanBlocks(walletID int) error {
+
+	wallet := mw.WalletWithID(walletID)
+	if wallet == nil {
+		return errors.E(ErrNotExist)
+	}
+
+	netBackend, err := wallet.internal.NetworkBackend()
+	if err != nil {
+		return errors.E(ErrNotConnected)
+	}
+
+	if mw.IsRescanning() || !mw.IsSynced() {
+		return errors.E(ErrInvalid)
+	}
+
+	go func() {
+		defer func() {
+			mw.syncData.mu.Lock()
+			mw.syncData.rescanning = false
+			mw.syncData.mu.Unlock()
+		}()
+
+		mw.syncData.mu.Lock()
+		mw.syncData.rescanning = true
+		mw.syncData.mu.Unlock()
+
+		if mw.blocksRescanProgressListener != nil {
+			mw.blocksRescanProgressListener.OnBlocksRescanStarted()
```

Comment:
This does not broadcast the ID for the wallet currently being rescanned. On an app with multiple wallets, how does the app know the wallet for which this `OnBlocksRescanStarted` notification is triggered? Same applies to the `OnBlocksRescanEnded` notification.